pkg/deviceshifu/deviceshifulwm2m/deviceshifulwm2m.go

```diff
@@ -0,0 +1,244 @@
+package deviceshifulwm2m
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifulwm2m/lwm2m"
+	"github.com/edgenesis/shifu/pkg/deviceshifu/utils"
+	"github.com/edgenesis/shifu/pkg/logger"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/edgenesis/shifu/pkg/k8s/api/v1alpha1"
+)
+
+// DeviceShifuLwM2M deviceshifu for LwM2M
+type DeviceShifuLwM2M struct {
+	server              *lwm2m.Server
```

Comment:
add some comment for this variable? what is this server's purpose, what this server do?